app/utils/results.py

```diff
@@ -223,3 +223,90 @@ def add_ip_card(html_soup: BeautifulSoup, ip: str) -> BeautifulSoup:
         # Inserting the element
         ref_element.insert_before(ip_tag)
     return html_soup
+
+
+def check_currency(response: str):
+    """Check whether the results have currency conversion
+
+    Args:
+        response: Search query Result
+
+    Returns:
+        list
+
+    """
+    soup = BeautifulSoup(response, 'html.parser')
+    a = soup.find(class_='ZINbbc xpd O9g5cc uUPGi')
+    tag = a.find('a')
+    if tag and tag['href'] == 'https://g.co/gfd':
+        currency1 = soup.find(class_="xUrNXd UMOHqf")
+        currency2 = soup.select_one('.kCrYT'
+                                    ' .BNeawe.iBp4i.AP7Wnd '
+                                    '.BNeawe.iBp4i.AP7Wnd')
+        currency1 = currency1.text.rstrip('=').split(' ', 1)
+        currency2 = currency2.text.split(' ', 1)
+        return [float(currency1[0]), currency1[1],
+                float(currency2[0]), currency2[1]]
```

Comment:
Could be better as a dict, so that these at least can be referenced by name:

```suggestion
        return {
            'currencyValue1': float(currency1[0]),
            'currencyLabel1': currency1[1],
            ...
        }
```

or something like that. I generally try to make variable names as descriptive as possible in cases like these.

Reply:
Yes. I will switch to a dictionary.